src/JoomlaBrowser.php

```diff
@@ -145,6 +145,61 @@ public function installJoomla()
         // Wait while Joomla gets installed
         $this->debug('I wait for Joomla being installed');
         $I->waitForText('Congratulations! Joomla! is now installed.', 60, ['xpath' => '//h3']);
+    }
+
+    /**
+     * Install Joomla removing the Installation folder at the end of the execution
+     */
+    public function installJoomlaRemovingInstallationFolder()
+    {
+        $I = $this;
+
+        $I->installJoomla();
```

Comment:
Exactly!, I have done it to keep backwards compatibility and to keep DRY. The drawback is that we are back to the default function of not removing installation folder. Witch will crash in these days when right before the release this parameter is changed to stable (requiring to remove the installation folder before being able to access to the administrator): https://github.com/joomla/joomla-cms/blob/staging/libraries/cms/version/version.php#L28-L29

We will have to change in the current repos using JoomlaBrowser to installJoomlaRemovingInstallationFolder to avoid crashing, but is a issue that happens few times a year, so not in a hurry.